curvesim/pipelines/simple/trader.py

```diff
@@ -110,7 +110,7 @@ def post_trade_price_error(dx, coin_in, coin_out, price_target):
 
     trades = []
 
-    for pair in prices.index:
+    for pair in prices.keys():
```

Comment:
isn't this why we switched to native types? ;)  Cuz we can just do `pair in prices`.